frontend/src/scenes/hog-functions/email-templater/EmailTemplater.tsx

```diff
@@ -16,6 +17,7 @@ function EmailTemplaterForm({ mode }: { mode: 'full' | 'preview' }): JSX.Element
 
     const { featureFlags } = useValues(featureFlagLogic)
     const isMessagingTemplatesEnabled = featureFlags[FEATURE_FLAGS.MESSAGING_LIBRARY]
+    const unlayerEditorProjectId = posthog.getFeatureFlagPayload(FEATURE_FLAGS.UNLAYER_EDITOR_PROJECT_ID)
```

Comment:
Interesting idea to to do this via a feature flag payload 🤔 I've usually seen us do this via the "app context" approach passed from django. But I guess this works...

My only concern is it makes it a little harder to keep track of all our keys as it isn't IaC. Maybe in this case it would be better to stick with injecting it the more "usual" way?

Reply:
For FOSS it would just be blank though 🤔 (whilst allowing the user to set it if they pleased).

No strong feelings here though